test/unit/gateways/quickpay_v10_test.rb

```diff
@@ -268,4 +272,14 @@ def expected_expiration_date
     '%02d%02d' % [@credit_card.year.to_s[2..4], @credit_card.month]
   end
 
+  def transcript
+   'POST /payments/7452604/authorize?synchronized HTTP/1.1\r\nContent-Type: application/json\r\nAuthorization: Basic OjAzNTA4ZTc3OTFiYTZjOWQwZTY4MzA3MTZlNjUwZjM1YzQzNDJjNGIzNTc2NzIzYWQ1NTZlMjM2Y2E0Yzc3ODg=\r\nUser-Agent: Quickpay-v10 ActiveMerchantBindings/1.52.0\r\nAccept: application/json\r\nAccept-Version: v10\r\nAccept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3\r\nConnection: close\r\nHost: api.quickpay.net\r\nContent-Length: 136'
```

Comment:
@anellis Looks like this test will only be using the final string in the method.  Using `%q` like this: https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/nmi_test.rb#L421 would be :egg: cellent